modules/rosa-sdpolicy-am-aws-compute-types.adoc

```diff
@@ -379,8 +410,22 @@ Support for the GPU instance type software stack is provided by AWS. Ensure that
 - i4i.12xlarge (48 vCPU, 384 GiB)	 
 - i4i.16xlarge (64 vCPU, 512 GiB)	 
 - i4i.24xlarge (96 vCPU, 768 GiB) 
-- i4i.32xlarge (128 vCPU, 1024 GiB)	 
-- i4i.metal (128 vCPU, 1024 GiB)
+- i4i.32xlarge (128 vCPU, 1,024 GiB)	 
+- i4i.metal (128 vCPU, 1,024 GiB)
+- m5ad.xlarge (4 vCPU, 16 GiB)
+- m5ad.2xlarge (8 vCPU, 32 GiB)
+- m5ad.4xlarge (16 vCPU, 64 GiB)
+- m5ad.8xlarge (32 vCPU, 128 GiB)
+- m5ad.12xlarge (48 vCPU, 192 GiB)
+- m5ad.16xlarge (64 vCPU, 256 GiB)
+- m5ad.24xlarge (96 vCPU, 384 GiB)
+- m5d.xlarge (4 vCPU, 16 GiB)
+- m5d.2xlarge (8 vCPU, 32 GiB)
+- m5d.4xlarge (16 vCPU, 64 GiB)
+- m5d.8xlarge (32 vCPU, 28 GiB)
+- m5d.12xlarge (48 vCPU, 192 GiB)
+- m5d.16xlarge (64 vCPU, 256 GiB)
+- m5d.24xlarge (96 vCPU, 384 GiB)
 
 &#8224; This instance type provides 72 logical processors on 36 physical cores.
```

Comment:
You changed the language in the general section from provide to offer. You may want to match throughout.